src/Psalm/Internal/Analyzer/Statements/Expression/Call/FunctionCallAnalyzer.php

```diff
@@ -236,6 +236,13 @@ public static function analyze(
                 strtolower($function_id)
             );
 
+            ArgumentMapPopulator::recordArgumentPositions(
+                $statements_analyzer,
+                $stmt,
```

Comment:
Argument 2 of Psalm\Internal\Analyzer\Statements\Expression\Call\ArgumentMapPopulator::recordargumentpositions expects PhpParser\Node\Expr\MethodCall|PhpParser\Node\Expr\StaticCall, PhpParser\Node\Expr\FuncCall provided
```
                $stmt,
                ^^^^^
```